mast/templates/login.html

```diff
@@ -1,10 +1,44 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-<h1>HELLo there</h1>
-</body>
-</html>
\ No newline at end of file
+{% extends "auth_template.html" %}
+{% block form %}
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-md-4 ml-4">
+            <img src="{{url_for('static', filename='pics/running.svg')}}" alt="athlete" style="max-width: 100%; max-height: 100%;" >
+        </div>
+        <div class="col-md-2">
+        </div>
+        <div class="col-md-3">
+            <form class="form-group">
+                <div class="text-center mb-4">
+                    <h1 class="h2 mb-3">
+                        Matfyz<br>
+                        Activity<br>
+                        <span class="primary_header">Sport<br>Tracker</span>
+                    </h1>
+                    <p>Compete, track and enjoy</p>
+                </div>
+
+                <div class="form-label-group">
+                    <input type="email" id="inputEmail" class="form-control" placeholder="Email address" required
+                        autofocus>
+                    <label for="inputEmail">Email address</label>
+                </div>
+
+                <div class="form-label-group">
+                    <input type="password" id="inputPassword" class="form-control" placeholder="Password" required>
+                    <label for="inputPassword">Password</label>
+                </div>
+
+                <div class="checkbox mb-3">
+                    <label>
+                        <input type="checkbox" value="remember-me"> Remember me
+                    </label><br>
+                    <a href="{{ url_for( 'register' ) }}">Sign-up</a>
```

Comment:
I'd rather avoid using "Sign in" and "Sign up" - could be confusing a little bit :)

Suggestion: "Create a new account" / "Are you new?"